test/index.js

```diff
@@ -201,6 +212,24 @@ test('off()', async t => {
 	t.deepEqual(calls, [1]);
 });
 
+test('off() - multiple event names', async t => {
+	const emitter = new Emittery();
+	const calls = [];
+	const listener = () => calls.push(1);
```

Comment:
```suggestion
	const listener = () => {
		calls.push(1);
	};
```

Don't use inline arrow function if it doesn't actually return anything. Applies in many other places too.

Reply:
I also changed it in other places that are not related to this PR, I hope that is ok.